gcloud/bigquery/table.py

```diff
@@ -50,6 +83,32 @@ def path(self):
         return '%s/tables/%s' % (self._dataset.path, self.name)
 
     @property
+    def schema(self):
+        """Table's schema.
+
+        :rtype: list of ``SchemaField``
+        :returns: fields describing the schema
+        """
+        return list(self._schema)
+
+    @schema.setter
+    def schema(self, value):
+        """Update table's schema
+
+        :type value: list of ``SchemaField``
+        :param value: fields describing the schema
+
+        :raises: TypeError if 'value' is not a sequence, or ValueError if
+                 any item in the sequence is not a SchemaField
+        """
+        fields = list(value)
+        if len(fields) > 0:
+            types = set([type(field) for field in fields])
+            if types != set([SchemaField]):
+                raise ValueError('Schema items must be fields')
+        self._schema = tuple(value)
```

Comment:
Why cast to `list` at beginning and then to `tuple` here? (I understand the tuple cast, just not the first one.)